dash-pipeline/bmv2/dash_pipeline.p4

```diff
@@ -300,7 +300,12 @@ control dash_ingress(
         routing_action_apply.apply(hdr, meta);
 
         /* Underlay routing */
-        meta.dst_ip_addr = (bit<128>)hdr.u0_ipv4.dst_addr;
+        if (meta.routing_actions & dash_routing_actions_t.ENCAP_U1 != 0) {
```

Comment:
Hi Junhua, do you mind to help add a small comment here that explains the dst_ip_addr will be used as key for underlay routing lookup?

Reply:
Updated the comment